portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,22 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let index = 0;
+function showNext() {
+    index = index +1;
+    if(index ==2){
+        const imgUrl = 'Me.jpg';
+        const imgElement = document.createElement('img');
+        imgElement.width = '550';
+        imgElement.height= '450';
+        imgElement.src = imgUrl;
+        const imageContainer = document.getElementById('picture');
+        imageContainer.appendChild(imgElement);
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    }
+    if(index == 1){
+        const summaryContainer = document.getElementById('summary');
+        summaryContainer.innerText="Currently, I'm a 2nd year student at the Rose-Hulman Institute of Technology working towards a bachelor in Computer Science. I grew up in Louisville, Kentucky and moved on to college early. I completed two years at Western Kentucky University while simultaneously obtaining my highschool diploma. The summer of my graduation I was accepted as a Google CSSI student and spent 3 weeks at the Google Cambridge Office. A year later and I am now working as a virtual Google intern in my hometown of Louisville."
```

Comment:
JS has a line length limit, so you'll need to break this up with `+`. Also, it would be nice to declare the text as a constant so that you can just do something like `summaryContainer.innerText = bioText;` here. It's easier to read code when you aren't distracted by content like long text.

https://google.github.io/styleguide/jsguide.html#formatting-column-limit